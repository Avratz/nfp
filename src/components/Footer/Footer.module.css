.footer {
  background-color: var(--neutral-2);
  margin: auto;
  padding: 4rem 0 1.5rem;
}

.logoContainer {
  padding-left: 2rem;
  padding-right: 2rem;
}

.heading {
  font-family: var(--font-poppins) !important;
  font-weight: var(--font-regular) !important;
  margin-top: 2rem;
  margin-bottom: 1.5rem;
}

.createdBy {
  font-family: var(--font-poppins);
  font-weight: var(--font-regular);
  font-size: 0.75rem;
  text-align: center;
  color: var(--neutral-4);
  padding-top: 1.5rem;
  padding-bottom: 1.5rem;
}

@media screen and (min-width: 768px) {
  .logoContainer {
    display: flex;
    flex-direction: column;
    align-items: center;
  }
}
