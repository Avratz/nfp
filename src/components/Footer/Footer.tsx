import { Logo } from '@/components/Logo/Logo'
import { Container } from '@/ui/Container/Container'
import { Divider } from '@/ui/Divider/Divider'
import { Heading } from '@/ui/Heading/Heading'

import styles from './Footer.module.css'

export function Footer() {
  return (
    <footer className={styles.footer}>
      <Container padding={false}>
        <div className={styles.logoContainer}>
          <Logo />
          <Heading as="h2" className={styles.heading}>
            The New Creative Economy
          </Heading>
        </div>
        <Divider />
        <p className={styles.createdBy}>Created with ❤️ by Avratz</p>
      </Container>
    </footer>
  )
}
