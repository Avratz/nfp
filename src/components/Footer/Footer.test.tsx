import { render } from '@testing-library/react'
import { Footer } from './Footer'

jest.mock('@/assets/images/paisanos-flag.svg', () => {
  const PaisanosFlag = () => null
  PaisanosFlag.displayName = 'PaisanosFlag'
  return PaisanosFlag
})

describe('Footer', () => {
  it('renders Footer', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
