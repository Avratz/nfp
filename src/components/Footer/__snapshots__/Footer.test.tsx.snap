// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="logoContainer"
      >
        <a
          class="link logo"
          href="/"
        >
          <picture
            class="isotipo"
          />
          <h1
            class="heading 2xl inlineBlock heading"
          >
            NFPaisanos
          </h1>
        </a>
        <h2
          class="heading base heading"
        >
          The New Creative Economy
        </h2>
      </div>
      <hr
        class="divider horizontal"
      />
      <p
        class="createdBy"
      >
        Created with ❤️ by Avratz
      </p>
    </div>
  </footer>
</div>
`;
